app/assets/javascripts/show-free-text-question.js

```diff
@@ -0,0 +1,31 @@
+window.GOVUKAdmin = window.GOVUKAdmin || {};
+window.GOVUKAdmin.Modules = window.GOVUKAdmin.Modules || {};
+
+(function (Modules) {
+  "use strict";
+
+  Modules.ShowFreeTextQuestion = function () {
+    this.start = function($element) {
+      function addEventListener() {
+        var $yesRadio = $element.find('.yes-radio');
+        var $noRadio = $element.find('.no-radio');
+        var $textBox = $element.find('.free-text-question');
+
+        if ($yesRadio.length) {
+          $yesRadio.click(function(e) {
+            $textBox.removeClass('if-js-hide');
```

Comment:
I personally find double-negatives hard to read - since "remove" and "hide" are both negative - it might be nice to move this into a one-line function called `showTextBox` or similar? Then you could also make this line read a bit more fluently, like:

```javascript
$yesRadio.click(showTextBox);
```